source/administration/changelog.md

```diff
@@ -4,11 +4,233 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- In mobile view, Manage Members menu option no longer reads "View Members" for channel admins.
+- Fixed vertical space between posts in the right-hand sidebar.
+- Fixed flagged posts and recent mentions icon sizing in the header.
+- Fixed vertical space below link previews.
+- Usernames with dots now get mention notifications when followed by a comma or other symbol.
+- Deactivated users are no longer listed in the "Manage Members" modal.
+- Collapsable Account Setting menus now properly open in iOS Safari and Chrome browsers.
+- Removing an expired license now removes the blue bar header message.
+- "Next" button in More Channels list now takes you to the top of the next page, instead of the bottom.
+- Blue bar "Preview Mode" header message now disappears after enabling email notifications.
+- Full name is now editable in Account Settings if the first and last name attributes are not specified in **System Console > Authentication > LDAP**.
+- Added a back button to pinned posts list on the right-hand sidebar.
+- "Pinned" icon no longer overlaps text on consecutive posts or replies that hvae Markdown headings.
+- Uploading a profile picture on iOS no longer throws an error.
+- Fixed group message names in channel switcher (CTRL/CMD+K) for group messages not in your sidebar.
+- Channel notification preferences no longer appear saved when clicking Cancel.
+
+### Compatibility  
+
+#### Breaking changes:
+
+- If you're using NGINX as a proxy for the Mattermost Server, replace the `location /api/v3/users/websocket {` line with `location ~ /api/v[0-9]+/(users/)?websocket$ {` in the `/etc/nginx/sites-available/mattermost` NGINX configuration file. [See documentation to learn more](https://docs.mattermost.com/install/install-ubuntu-1404.html#configuring-nginx-as-a-proxy-for-mattermost-server).
+
+#### Removed and deprecated features
+- All APIv3 endpoints to be removed in November/2017 release (six months after APIv4 endpoints have gone stable).
+
+For a list of past and upcoming deprecated features, [see our website](https://about.mattermost.com/deprecated-features/).
+
+#### config.json   
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json` or the System Console.
+
+**Changes to Team Edition and Enterprise Edition**:
+
+ - Under `ServiceSettings` in `config.json`:
+   - Added `"EnablePostSearch": true` to control whether full text search queries are enabled. Disabling can lead to higher performance in large deployments.
+   - Added `"EnableUserStatuses": true` to control whether user statuses are shown in the web user interface. Disabling can lead to higher performance in large deployments.
+
+ - Under `SqlSettings` in `config.json`:
+   - Added `"DataSourceSearchReplicas": []` to to isolate searches to specific read-replicas for full text search queries for higher performance.
```

Comment:
>for full text search queries

Makes it sound like there are non full-text queries and that they are not affected by this setting.